packages/esm-active-visits-app/src/index.ts

```diff
@@ -16,6 +16,18 @@ export function startupApp() {
   defineConfigSchema(moduleName, configSchema);
 }
 
+export const homePageTilesSlot = getAsyncLifecycle(() => import('./home-page-tiles/metrics-slot.component'), options);
```

Comment:
Is there still a need for this? the active visits tiles can be added directly to the metric tiles slot already defined in `esm-home`